platform/app/src/net/gpii/app/GpiiActivity.java

```diff
@@ -49,20 +112,223 @@
+            WindowManager.LayoutParams.TYPE_SYSTEM_ALERT | 
+            WindowManager.LayoutParams.TYPE_PHONE |
+            WindowManager.LayoutParams.TYPE_SYSTEM_OVERLAY,
+            WindowManager.LayoutParams.FLAG_NOT_FOCUSABLE |
+            WindowManager.LayoutParams.FLAG_NOT_TOUCH_MODAL |
+            WindowManager.LayoutParams.FLAG_WATCH_OUTSIDE_TOUCH,
+            PixelFormat.TRANSLUCENT);
+
+        LayoutInflater inflater = (LayoutInflater) getSystemService(LAYOUT_INFLATER_SERVICE);
+        progressView = inflater
+            .inflate(R.layout.progress_download_layout, null);
+
+        installationButton = (Button)findViewById(R.id.installButton);
+        downloadButton = (Button) findViewById(R.id.downloadButton);
+
+        gpiiInfo = (RelativeLayout) findViewById(R.id.gpii_Info);
+
+        file = new File(Environment.getExternalStorageDirectory(), "gpii");
+
+        AssetManager assetManager = getResources().getAssets();
+
+        try {
+            XmlPullParserFactory factory = XmlPullParserFactory.newInstance();
+            factory.setValidating(false);
+            XmlPullParser myxml = factory.newPullParser();
+
+            InputStream raw = assetManager.open("GpiiDataResources.xml");
+            myxml.setInput(raw, null);
+            int eventType = myxml.getEventType();
+        
+            while(eventType != XmlPullParser.END_DOCUMENT) {
+                if(eventType == XmlPullParser.TEXT) {
+                
+                    switch(myxml.getLineNumber()){
+                        case 2:
+                            uriTar = myxml.getText();
+                            break;
+                        case 3:
+                            uriUserListeners = myxml.getText();
+                            break;
+                        case 4:
+                            gpiiCompatibleAndroidDevicesUrl = myxml.getText();
+                            break;
+                        case 5:
+                            gpiiRootDevicesUrl = myxml.getText();
+                            break;
+                        case 6:
+                            gpiiJS = myxml.getText();
+                            break;
+                        case 7:
+                            gpiiAPK = myxml.getText();
+                            break;
+                        case 8:
+                            gpiiUserListenersAPK = myxml.getText();
+                            break;
+                        default:
+                            Log.i(TAG,"Not suported XML entry");
+                            break;
+                    }
+                }
+                eventType = myxml.next();
+            }
+        } catch (XmlPullParserException e) {
+            e.printStackTrace();
+        } catch (IOException ioe) {
+            ioe.printStackTrace();
+        }
+    
+        try {
+            tarIncluded=true;
+            Arrays.asList(getResources().getAssets().list("")).contains("gpii-android.tar.gz");
```

Comment:
As far as I can see, the resource is called "gpii-android" (without the .tar.gz extension), and I'm not seeing any renaming-related process within this last commit. Am I overlooking it?